moped-editor/src/utils/test/dateAndTime.test.js

```diff
@@ -54,3 +55,39 @@ describe("makeFullTimeFromTimeStampTZ()", () => {
     expect(formattedTime).toBe("11:14:45 AM");
```

Comment:
these time-specific tests fail for me when i run them from Eastern Time :/  

looks like you can [set the timezone in a Jest config](https://stackoverflow.com/a/56482581/2103784) but i did wander down this path

Reply:
Glad that you tripped over this. Didn't realize that would be the case and thanks for pointing to the Jest config 🙏